.github/actions/flaky-tests-report/action.yml

```diff
@@ -0,0 +1,70 @@
+name: Flaky test report
+description: Extract flaky tests from PW test reports
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 20
+
+    - name: Install CTRF package globally
+      shell: bash
+      run: npm install -g ctrf
```

Comment:
Prefer installing `ctrf` from a lock file (such as `package.json` + `package-lock.json` using `npm ci` command) to prevent mutating dependencies (avoids the workflow suddenly breaking or compromised/malicious dependency from being used)

Reply:
I wanted to do that, but this package requires node 20, and the dashboard still uses 18

Reply:
One way you can do this is to put that inside the `.github/actions/flaky-tests-report` directory, the `cd` it in the action + `npm ci` (it's even better because it prevents installing/adding unneeded dependencies into the dashboard)